README.md

```diff
@@ -3,13 +3,41 @@
 PyCon India Blog (https://in.pycon.org/blog/)
 
 
-## Getting Started 
+## Getting Started
 
-Run the following commands in a virtualenv:
+
+
+Run the following commands to create virtualenv and start working:
+
+### Install pipenv
+
+Just use pip:
+
+```
+$ pip install pipenv
+```
+
+Or, if you’re using Ubuntu 17.10:
+
+```
+$ sudo apt install software-properties-common python-software-properties
+$ sudo add-apt-repository ppa:pypa/ppa
+$ sudo apt update
+$ sudo apt install pipenv
+```
+
+
+Otherwise, if you’re on MacOS, you can install Pipenv easily with Homebrew:
+
+```
+$ brew install pipenv
+```
+
+### Install dependencies and start server
 
 ```
-$ pip install -r requirements.txt
-$ ./develop_server.sh start
+$ pipenv install
+$ pipenv run ./develop_server.sh start
```

Comment:
Does this activate the virtual env ?

Reply:
`pipenv shell` will activate the virtual env but, since we have to run just bash script, we are doing `pipenv run [command]`. 
Although, you can run:
```
$ pipenv shell
$ ./develop_server.sh start
```
but it is same.
Should I change it?